doc/man/scons.xml

```diff
@@ -2918,16 +2918,22 @@ target file's directory.</para>
 <programlisting language="python">
 # scons will change to the "sub" subdirectory
 # before executing the "cp" command.
-env.Command('sub/dir/foo.out', 'sub/dir/foo.in',
-            "cp dir/foo.in dir/foo.out",
-            chdir='sub')
+env.Command(
```

Comment:
I'd avoid always using the arg names, common usage is to exclude target, source, action.

Reply:
it's a lot more typing and given we don't have any autocomplete editor support. I think it's better without. The manpage lists the order of arguments..  It's honestly rarely yielded questions on mailing list, irc, or discord.  target, sources, FLAGS for all builders.

Reply:
I'd say change these back.. Maybe somewhere we should say explicitly all builders take arguments in this order target, source,... but for now I think status quo on this is better.

Reply:
There is such wording in the beginning of the builder section.